pkg/kind/cluster.go

```
@@ -108,11 +110,23 @@ func (c *Cluster) getConfig() ([]byte, error) {
 		}
 	}
 
+	var registryConfig string = func() string {
+		for _, s := range c.registryConfig {
+			path := os.ExpandEnv(s)
+			println(path)
+			if _, err := os.Stat(path); err == nil {
+				return path
```

If an error is returned, this ignore the error correct?